.expeditor/Version_inspector/update_prod_list.sh

```diff
@@ -0,0 +1,34 @@
+##to update the prod_list.json 
```

Comment:
this file is fully commented out, do we want to commit it?

Reply:
update_prod_list.sh - This script is to generate the product list and regular expression. Which is not necessary to run all the time. Hence I had commented.
 I will uncomment it . I have noted in README anyway.